worldedit-core/src/main/java/com/sk89q/worldedit/math/transform/AffineTransform.java

```diff
@@ -165,17 +165,17 @@ public AffineTransform inverse() {
         double det = this.determinant();
         return new AffineTransform(
                 (m11 * m22 - m21 * m12) / det,
-                (m21 * m01 - m01 * m22) / det,
+                (m02 * m21 - m22 * m01) / det,
```

Comment:
this `m02` is literally the only change you made, can you revert all the other multiplications to their original order to shrink the diff?

Reply:
yeh

Reply:
I think he meant all of them, including this line.
Essentially you're only changing a 1 to a 2 and the diff should reflect that, so people dont have to play "where's waldo" to see what's going on :)